python/rmm/_lib/lib.pxd

```diff
@@ -65,7 +65,7 @@ cdef extern from "rmm/rmm.h" nogil:
         PoolAllocation = 1,
         CudaManagedMemory = 2,
 
-    cdef cppclass rmmOptions_t:
+    ctypedef struct rmmOptions_t:
```

Comment:
Doesn't this need to be a cppclass in order to get the default value of `devices`?

Reply:
It's still a struct in the `rmm_api.h` so I followed suit. We aren't dependent on default value of `devices` because we'll always pass a value as opposed to relying on the default constructor.

Reply:
I'm not sure if because it's a struct the default constructor for the `devices` vector gets called or not, @shwina?

Reply:
@kkraus14 in C++ structs *are* classes. The only difference is members default to public unless otherwise specified.  So they do have default constructors.

Reply:
Yeah, it has to be a `cppclass`. I think this bit us before somewhere.